impl/data.js

```diff
@@ -2,17 +2,23 @@
 'use strict';
 
 (() => {
-
   const os = require('os');
+  const fs = require('fs');
   const path = require('path');
+  const dbPath = process.env.DBDIR || path.join(os.homedir(), 'data');
```

Comment:
Awesome, thanks! Nitpicks:
1. Could we name it `GHUSER_DBDIR` instead? A bit like other env vars we have like `GITHUB_*` and `AWS_*`.
2. Can we mention it in the readme? There is already a mention to this code line. We could get rid of it and mention this env var instead :)

Reply:
Sorry I didn't realize that I had push permissions here. Just did it myself, thanks for the contribution :heart: